configs/db/postgres/000001_configuration-management.up.sql

```diff
@@ -53,12 +53,63 @@ CREATE TABLE config_cachedeps (
 
 CREATE INDEX config_cachedeps_idx_cacheid ON config_cachedeps ("cache_id");
 
---DDL for cluster properties:
-CREATE TABLE cluster_props (
-	"id" SERIAL UNIQUE, --just another unique identifer for a property-entry
+--DDL for clusters:
+CREATE TABLE clusters (
+     id uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
 	"cluster" text NOT NULL,
-	"key" text NOT NULL,
-	"value" text NOT NULL,
-	"created" TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'utc'),
-	CONSTRAINT cluster_metadata_pk PRIMARY KEY ("cluster", "key")
+	"runtime_name" text,
+	"runtime_description" text,
+    "global_account_id" text,
+    "sub_account_id"    text,
+    "service_id"       text,
+    "service_plan_id"   text,
+    "shoot_name"       text,
+    "instance_id"      text,
+	"created" TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'utc')
+);
+
+CREATE TABLE configurations (
+     id uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
```

Comment:
As the configuration will be immutable, it could be helpful to use incremental IDs instead of UUIDs. They would have the benefit, that they implicitly reflect the timely order of confiurations.